service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/modifier/ManageOrdersCaseDataFixer.java

```diff
@@ -48,4 +50,16 @@ public CaseData fix(CaseData caseData) {
 
         return caseData;
     }
+
+    public CaseDetails fixAndRetriveCaseDetails(CaseDetails caseDetails) {
+        if (!isNull((caseDetails.getData().get("manageOrdersOperation")))) {
+            String operation = caseDetails.getData().get("manageOrdersOperation").toString();
```

Comment:
If you want you could probably store this before the outer if without the toString call, might make things shorter. The toString can then be added in the equal if you still want it